templates.go

```diff
@@ -55,12 +55,14 @@ func initTemplate(name string) {
 			filepath.Join(templatesDir, name+".tmpl"),
 			filepath.Join(templatesDir, "include", "posts.tmpl"),
 			filepath.Join(templatesDir, "include", "footer.tmpl"),
+			filepath.Join(templatesDir, "include", "render.tmpl"),
 			filepath.Join(templatesDir, "base.tmpl"),
 		))
 	} else {
 		templates[name] = template.Must(template.New("").Funcs(funcMap).ParseFiles(
 			filepath.Join(templatesDir, name+".tmpl"),
 			filepath.Join(templatesDir, "include", "footer.tmpl"),
+			filepath.Join(templatesDir, "include", "render.tmpl"),
```

Comment:
We don't want this included in every templated page, but we do need it if `name == "collection" || name == "collection-tags" || name == "collection-post" || name == "post"`

I think the best way to do this will be passing a `[]string` to a single `ParseFiles()` call, where that array is constructed based on which template is being initialized. I'll make this change when I review this a little later.